packages/manager/apps/pci-instances/src/pages/instances/create/hooks/useGetForm.ts

```diff
@@ -0,0 +1,46 @@
+import { deps } from '@/deps/deps';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useForm } from 'react-hook-form';
+import { nameDefaultValue } from '../components/Name.component';
+import { quantityDefaultValue } from '../components/QuantitySelector.component';
+import { instanceCreationSchema } from '../CreateInstance.page';
+import { selectContinent } from '../view-models/selectContinents';
+import { selectDeploymentModes } from '../view-models/selectDeploymentMode';
+import { selectLocalizations } from '../view-models/selectLocalizations';
+import { TDeploymentMode } from '@/types/instance/common.type';
+
+export const useGetForm = (projectId: string) => {
```

Comment:
I'm not confident with the `useGet..` as it's too much related to a CRUD operation.

Could we say this hook is much more a `useForm` or `useFormMethods` ?